### ISSUE TITLE
All versions after 5.0.1 not working

### ISSUE THREAD

**Issue by riberman** [orkestral/venom#2343]
## Description
I using the version **5.0.1** and the correction like this [issue](https://github.com/orkestral/venom/issues/2314) in file **node_modules/venom-bot/dist/lib/wapi/wapi.js** working fine.

But all versions after **5.0.1** not working, specifically in versions **5.0.6** and **5.0.7**. The other **5.0.2**, **5.0.3**, **5.0.4** and **5.0.5** have even more issues in log.

When start the server and try read session QR code the message noOpenBrowser appers in log.

I clean all old sessions, removed **node_modules** folder and **package-lock.json** and run again **npm i** but not resolves.
## Environment

- **Venom version(s):** [5.0.6, 5.0.7]
- **OS:** [Ubuntu 20.04.5 LTS]
- **Node version:** [Node 18.16.1]

## Steps to Reproduce

1. [Start the server]
2. [Read QR code comand]

## Log Output

```
0|venom    | - headless option is active, browser hidden
0|venom    | - Page successfully accessed
0|venom    | - waiting for introduction
0|venom    | - [instance: teste]: Waiting for QRCode Scan: Attempt 47
0|venom    | - Waiting... checking the browser...
0|venom    | - Error no open browser....
0|venom    | Error no open browser....
```

**Comment by ghayman** [comment]
Please try

npm install orkestral/venom

this will install 5.0.7 directly from Github - I am not convinced that that the npm package is upto date.

I was told this was working but I'm not too sure... anyway, please give this a go - let us know.

**Comment by MatheusKonradXavier** [comment]
> Please try
> 
> npm install orkestral/venom
> 
> this will install 5.0.7 directly from Github - I am not convinced that that the npm package is upto date.
> 
> I was told this was working but I'm not too sure... anyway, please give this a go - let us know.

This resolved my problem, thanks

**Comment by riberman** [comment]
> Please try
> 
> npm install orkestral/venom
> 
> this will install 5.0.7 directly from Github - I am not convinced that that the npm package is upto date.
> 
> I was told this was working but I'm not too sure... anyway, please give this a go - let us know.

This worked for me, version 5.0.7 running.

**Comment by GustavoTG75** [comment]
> 
I tried npm install orkestral/venom. But unfortunately it didn't work on my PC. This is the error that I got:

npm ERR! code ENOENT
npm ERR! syscall spawn git
npm ERR! path git
npm ERR! errno -4058
npm ERR! enoent An unknown git error occurred
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent

npm ERR! A complete log of this run can be found in: C:\Users\gustavo.teixeira\AppData\Local\npm-cache\_logs\2023-06-23T17_20_23_413Z-debug-0.log
